packages/graphql-generator/src/models.ts

```diff
@@ -61,7 +62,7 @@ export async function generateModels(options: GenerateModelsOptions): Promise<Ge
   return Promise.all(
     appsyncLocalConfig.map(async config => {
       const content = await codegen(config);
-      return { [config.filename]: content };
+      return { [config.filename.split(path.win32.sep).join(path.posix.sep)]: content };
```

Comment:
This will change the output of `generateModels` so that the filepath will always use posix separators. IMO this should be the correct behavior so that the package is consistent across OS.